tools/sigh.js

```diff
@@ -536,6 +538,12 @@ function test(args) {
         mocha.suite.emit('require', null, '${test}', mocha);
         mocha.suite.emit('post-require', global, '${test}', mocha);
       `);
+      if (options.retries) {
```

Comment:
validate value?

also minimist docs say:

""Numeric-looking arguments will be returned as numbers unless opts.string or opts.boolean is set for that argument name."""

Reply:
I added validation just now to ensure that is a number. I tried adding a `console.log()` for non-numeric inputs, but logging it in `buildTestRunner()` would print it multiple times.